static/js/pages/instances/InstanceList.tsx

```diff
@@ -283,62 +302,65 @@ const InstanceList: FC = () => {
 
   return (
     <>
-      <aside className="l-toolbar">
-        <h4>
-          {visibleInstances.length}{" "}
-          {visibleInstances.length === 1 ? "Instance" : "Instances"}
-        </h4>
-        <div className="filters">
-          <SearchBox
-            className="search-box margin-right"
-            id="filter-query"
-            type="text"
-            onChange={(value) => {
-              setQuery(value);
-            }}
-            placeholder="Search"
-            value={query}
-          />
-          <Select
-            wrapperClassName="margin-right"
-            id="filter-state"
-            onChange={(v) => {
-              setStatus(v.target.value);
-            }}
-            options={[
-              {
-                label: "All statuses",
-                value: "any",
-              },
-              ...instanceStatuses,
-            ]}
-            value={status}
-          />
-          <Select
-            id="filter-type"
-            onChange={(v) => {
-              setType(v.target.value);
-            }}
-            options={[
-              {
-                label: "Containers and VMs",
-                value: "any",
-              },
-              ...instanceListTypes,
-            ]}
-            value={type}
-          />
-        </div>
-        <Button
-          appearance="positive"
-          onClick={() => panelParams.openCreateInstance(project)}
+      <main className="l-main instance-list">
+        <div
+          className={classnames("p-panel", {
+            "has-side-panel": !!panelParams.instance,
+          })}
         >
-          Create new
-        </Button>
-      </aside>
-      <main className="l-main">
-        <div className="p-panel">
-          <div className="p-panel__content p-instance-content">
+          <div className="p-panel__header instance-list-header">
+            <div className="instance-header-left">
+              <h4>
+                {visibleInstances.length}
+                &nbsp;
+                {visibleInstances.length === 1 ? "Instance" : "Instances"}
+              </h4>
+              <SearchBox
```

Comment:
Sorry, I missed that yesterday. Is there a way to add labels for all the filters here with the `u-off-screen` class?

Reply:
Good catch! I just added the aria-labels.